blueprints/secops/age/ops2deb.yml

```diff
@@ -0,0 +1,20 @@
+name: age
+matrix:
+  versions:
+    - 1.0.0
+    - 1.1.0
+    - 1.1.1
+  architectures:
+    - amd64
+    - arm64
+homepage: https://age-encryption.org/
+summary: age is a simple, modern and secure file encryption tool, format, and Go library
+description: |-
+  A simple, modern and secure encryption tool (and Go library) with small
```

Comment:
It is also recommended not to repeat the summary in the description:

`age features small explicit keys, no config options, and UNIX-style
 composability.`

Reply:
thx, fixed